openedx/core/lib/block_structure/transformers.py

```diff
@@ -107,22 +107,22 @@ def transform(self, block_structure):
         collection, in the order that the transformers were added.
         """
         self._transform_with_filter(block_structure)
-        self._full_transform(block_structure)
+        self._transform_without_filter(block_structure)
```

Comment:
I was thinking of these method names as active verbs: in order to `transform()`, needed sub-actions are `_transform_with_filter()` and `_transform_without_filter()`.